test/test_interface.idl

```diff
@@ -32,6 +32,7 @@ interface TestInterface {
   byte byteMethod(byte number);
   octet octetMethod(octet number);
   short shortMethod(short number);
+  unsignedLong unsignedlongMethod(unsignedLong number);
```

Comment:
Return type and parameter type should be ```unsinged long```.
Also, method name should be ```unsignedLongMethod```

Reply:
If I insert a white space between unsigned and long. it causes an error.
I have tried to solve this problem, but I failed.